### ISSUE TITLE
Update Readme

### ISSUE THREAD

**Issue by mseremet** [JesusLeon/fogio#3]
The README.md does not say anything on
* which custom fields are needed
* how to use the command line

**Comment by JesusLeon** [comment]
@mseremet indeed. Although currently is a one-command tool. All parameters are based on configurations within the classes (not the best way, i know). If really required, it can graduate to a proper (and small) CLI tool.

Need to update the README.
